solidity/contracts/BancorNetwork.sol

```diff
@@ -10,9 +10,14 @@ import './utility/interfaces/IContractFeatures.sol';
 import './utility/interfaces/IWhitelist.sol';
 import './token/interfaces/IEtherToken.sol';
 import './token/interfaces/ISmartToken.sol';
-import './token/interfaces/INonStandardERC20.sol';
 import './bancorx/interfaces/IBancorX.sol';
 
+// interface of older converters for backward compatibility
+contract ILegacyBancorConverter is IBancorConverter {

```

Comment:
`is IBancorConverter` is practically redundant, since no other `IBancorConverter` is used.

Reply:
@yudilevi : Fixing this myself.